src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/wrapper/ErrorHandlingWrapper.java

```diff
@@ -87,19 +87,18 @@ public Boolean wrapDocUploadFailure(Envelope envelope, Supplier<Boolean> supplie
         }
     }
 
-    public Boolean wrapDeleteBlobFailure(
+    public void wrapDeleteBlobFailure(
```

Comment:
Returning a boolean allows for easier check + chaining + would leave the signature consistent with the other similar methods. If you really care about this, go on, but in my opinion this is not an improvement.